plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -710,7 +701,6 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
             external = false;
         }
         Path targetPath = new Path(location);
-        ensurePathExists(session, targetPath);
```

Comment:
This check can be skipped because directory is created implicitly when first file is created.

Reply:
This should be part of the commit message